Library/ViewModels/LiveStreamChatViewModel.swift

```diff
@@ -204,6 +216,12 @@ LiveStreamChatViewModelOutputs {
       return AppEnvironment.current.currentUser != nil
     }
 
+    self.textFieldShouldChangeCharactersInRangeReturnProperty <~
+      textFieldShouldChangeCharactersInRangeProperty.signal.skipNil().map { currentString, range, string in
+        return (currentString as NSString)
+          .replacingCharacters(in: range, with: string).characters.count <= maxMessageLength
```

Comment:
True, pasted text over the limit would just do nothing 🤔 

I think `1` would be easiest and I could change the `x/x` to just be a number that decrements and then goes into negatives when it exceeded.